package.json

```diff
@@ -21,7 +21,6 @@
     "nodemon": "^1.18.7"
   },
   "dependencies": {
-    "cors": "^2.8.5",
```

Comment:
@sjmarsnc is there a reason you are removing this dependency? We need it for the sockets to work!